src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -308,9 +338,12 @@ public Path generate(Logger log) throws KaitaiException {
 		}
 
 		ProcBuilder builder = process(log)
-			.withArgs("--target", "java")
-			.withArgs("--outdir", output.toFile().getAbsolutePath())
-			.withArgs("--java-package", getPackageName());
+				.withArgs("--target", "java")
+				.withArgs("--outdir", output.toFile().getAbsolutePath())
+				.withArgs("--java-package", getPackageName());
+		if (isNoAutoRead()) {
+			builder.withArgs("--no-auto-read");
```

Comment:
Is this argument is supported by `ksc` version `0.8`?

Reply:
it's supported from 0.9, I need it to avoid creating a private _read method, and calling the read from the constructor.
because I need to override the _read method to handle some exception cases.